create-doks-with-terraform-flux/README.md

```diff
@@ -1 +1,470 @@
+            name: busybox
+        restartPolicy: Always
+    EOF
+
+    # busybox Kustomization yaml file
+    cat << EOF > "${APPS_PATH}/kustomization.yaml"
+    apiVersion: kustomize.config.k8s.io/v1beta1
+    kind: Kustomization
+
+    commonLabels:
+        app: busybox
+
+    namespace: busybox
+
+    resources:
+       - busybox.yaml
+       - busybox-ns.yaml
+    EOF
+
+    flux create kustomization "$FLUX_CD_KUSTOMIZE_NAME" \
+        --source="$FLUX_CD_SOURCE_NAME" \
+        --path="$APPS_PATH" \
+        --prune=true \
+        --validation=client \
+        --interval=5m \
+        --export > "./${TF_VAR_github_repository_target_path}/${FLUX_CD_KUSTOMIZE_NAME}-kustomization.yaml"
+
+    git add -A && git commit -am "Adding the Flux CD Kustomize component"
+    ```
+6. The final step would be to push the changes:
+
+    ```bash
+    git push origin $TF_VAR_github_repository_branch
+    ```
+
+## Inspecting the results
+
+After a while the busybox namespace and associated pod should be created and running. Let's see what flux has to say about it first:
+
+```bash
+flux get kustomizations
+```
+
+The output should be something similar to (notice the `busybox` line):
+
+```
+NAME       	READY	MESSAGE                                                        	REVISION                                     	SUSPENDED 
+busybox    	True 	Applied revision: main/fa69f917302bcfd35d2959ebc398b3aa13102480	main/fa69f917302bcfd35d2959ebc398b3aa13102480	False    	
+flux-system	True 	Applied revision: main/fa69f917302bcfd35d2959ebc398b3aa13102480	main/fa69f917302bcfd35d2959ebc398b3aa13102480	False 
+```
+
+Now let's see what Kubernetes has to say. Examine the namespaces first:
+
+```bash
+kubectl get ns
+```
+
+The output should be something similar to (notice the `busybox` line):
+
+```
+NAME              STATUS   AGE
+busybox           Active   30s
+default           Active   26h
+flux-system       Active   26h
+kube-node-lease   Active   26h
+kube-public       Active   26h
+kube-system       Active   26h
+```
+
+Where is our Pod? Let's find out:
+
+```bash
+kubectl get pods -n busybox
+```
+
+The output should be something similar to (notice the `busybox1` line):
+
+```
+NAME       READY   STATUS    RESTARTS   AGE
+busybox1   1/1     Running   0          42s
+```
+
+**Success!**
+
+# Final notes
+
+## Overriding default variables for Terraform
+
+The best way of doing it is to create a `project.tfvars` file. There's a [project.tfvars.sample](project.tfvars.sample) provided in this repo so we can just rename it:
+
+```bash
+cp project.tfvars.sample project.tfvars
+```
+And then fill in the right values for the project - [variables.tf](variables.tf) contains the description for each variable.
+
+Now we're going to use this file to override the desired input variables and update the infrastructure as seen below:
+
+```bash
+terraform plan -var-file="project.tfvars"
+terraform apply -var-file="project.tfvars"
+```
+
+## Testing other Flux CD features
```

Comment:
```suggestion
## Testing Other Flux CD features
```

Reply:
I don't quite understand why is `other` capitalised here. For me it looks kinda strange.